opentracing/tracer.py

```diff
@@ -38,6 +38,7 @@ def __init__(self):
 
     def start_span(self,
                    operation_name=None,
+                   child_of=None,
```

Comment:
admittedly unrelated (but an inconsistency, and I think we're better off with the way things are in the other languages).